test/parallel/test-runner-coverage.js

```diff
@@ -154,3 +140,56 @@ test('test spec coverage reporter', async (t) => {
     assert(!findCoverageFileForPid(result.pid));
   });
 });
+
+test('single process coverage is the same with --test', () => {
+  if (!process.features.inspector) {
+    return;
+  }
+
+  const fixture = fixtures.path('test-runner', 'coverage.js');
+  const args = [
+    '--test', '--experimental-test-coverage', '--test-reporter', 'tap', fixture,
+  ];
+  const result = spawnSync(process.execPath, args);
+  const report = getTapCoverageFixtureReport();
+
+  assert(result.stdout.toString().includes(report));
+  assert.strictEqual(result.stderr.toString(), '');
+  assert.strictEqual(result.status, 0);
+  assert(!findCoverageFileForPid(result.pid));
+});
+
+test('coverage is combined for multiple processes', () => {
+  if (!process.features.inspector) {
+    return;
+  }
```

Comment:
```suggestion
test('coverage is combined for multiple processes', { skip: !process.features.inspector ? 'inspector disabled' : false}, () => {
```
NIT, add a description

Reply:
I am pretty sure you are referring to `common.skip`:
https://github.com/nodejs/node/blob/3e7459055bbcfe35e203896cb9afffa10db2f256/test/common/index.js#L598-L601

Reply:
No. I'm pretty sure I tried using `skip` when I initially wrote this test. I'll try again to verify though.